python/src/aac/io/parser/_parse_source.py

```diff
@@ -43,24 +44,37 @@ def parse(source: str, source_uri: Optional[str] = None) -> list[Definition]:
 
     if linesep not in source:
         sanitized_source = sanitize_filesystem_path(source)
-        if path.lexists(sanitized_source):
+        if path.isfile(sanitized_source):
```

Comment:
lexists was giving false positives since an empty string passed in would become sanitized to the workspace directory -- causing the parser to try and parse the directory instead of treating it as an empty string.